src/peer.c

```diff
@@ -24,24 +24,11 @@
  * SOFTWARE.
  */
 
-#include <arpa/inet.h>
```

Comment:
Why have you taken out all these includes? I think the types defined in these headers are used in peer.c